docs/tutorials/podman.rst

```diff
@@ -17,36 +18,63 @@ Enabling Podman
+Once you have installed Podman and its dependencies on the platform of your choice, you'll need to make sure that the ``podman`` binary, usually installed as ``/usr/bin/podman``, is aliased to ``docker``.
+
+On some CentOS and Fedora releases, you can install a package named ``podman-docker`` to do this for you, but on other platforms, you'll need to take of this yourself.
+
+- To alias ``podman`` to ``docker``, you can simply run this command::
+
+    $ alias docker=podman
+
+.. note::
+   Running this command only makes a temporary alias. For a more permanent alias, you should place that command in your ``bashrc`` or equivalent file.
 
-- If ``$HOME/bin`` is in your ``$PATH``, you can create a symbolic link there::
+Getting docker-compose to work with Podman
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    ln -s $(which podman) $HOME/bin/docker
+To allow ``podman`` to work with ``docker-compose``, you'll need to enable a podman socket which pretends to be ``docker``.
 
-- If you want to instead make ``docker`` a system-wide alias for ``podman``, you can create your symlink in ``/usr/local/bin``, an action that normally requires ``root`` privileges::
+For rootless containers, this requires you to start the ``podman.service`` as a regular user and set the ``DOCKER_HOST`` environment variable. This can be done as follows::
 
-    sudo ln -s $(which podman) /usr/local/bin/docker
+  # To start the podman service
+  $ systemctl --user start podman.service
+
+  # To set the DOCKER_HOST environment variable
+  $ export DOCKER_HOST="unix://$XDG_RUNTIME_DIR/podman/podman.sock"
+
+.. note::
+   As with the previous ``alias`` command, if you'd like to make the ``DOCKER_HOST`` variable permanent, you should put the entire export command in your ``bashrc`` or equivalent file.
+
+Fixing SELinux Errors
+~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+   Disabling ``SELinux`` or setting it to *permissive mode* on your system is **highly discouraged and will render your system vulnerable.**
 
+If you're running on a system that has ``SELinux``, you might run into a few errors that cause rootless containers to not be able to read configuration files.
 
-Enabling podman-compose
-~~~~~~~~~~~~~~~~~~~~~~~
+Instead of disabling SELinux, you could instead apply the appropriate labels to the tutor root directory. This can be done as follows::
 
-``podman-compose`` is available as a package from PyPI, and can thus be installed with ``pip``. See `its README <https://github.com/containers/podman-compose/blob/devel/README.md>`_ for installation instructions. Note that if you have installed Tutor in its virtualenv, you'll need to run ``pip install podman-compose`` in that same virtualenv.
+  # Set the SELinux type of the tutor root directory and all of it's subdirectories to `container_file_t`
+  $ sudo semanage fcontext -a -t container_file_t "/home/tutor/.local/share/tutor(/.*)?"
```

Comment:
Replace "/home/tutor/.local/share/tutor" by "$(tutor config printroot)". And the note below can be removed.

Reply:
Good point! I'll fix that real quick.